controller_status_topic: /controller/status
arm_status_topic: /arm_controller/status
gripper_status_topic: /gripper/status

action_list:
  - move: [3.5, 0.6]
  - move: [3.0, 4.0]
  - move: [1.2, 3.0]